app/scripts/controllers/app-state-controller.ts

```diff
@@ -86,6 +86,7 @@ export type AppStateControllerState = {
   custodianDeepLink?: { fromAddress: string; custodyId: string };
   slides: CarouselSlide[];
   throttledOrigins: ThrottledOrigins;
+  upgradeSplashPageAcknowledgedForAccounts: string[];
```

Comment:
Minor, an interesting conceptual question if this is app state, or more a preference that the user is setting by interacting with the splash screen.

Reply:
I think it is not preference as user is not even aware that this exist.